### ISSUE TITLE
Plus button lacks screen reader support

### ISSUE THREAD

**Issue by untill** [NorskRegnesentral/ccc21#81]
TalkBack reads it like: "Knapp".

**Comment by untill** [comment]
I fixed it recently, but this has to be tested after deploy.
